app/controllers/users_controller.rb

```diff
@@ -27,6 +27,11 @@ def demote
     redirect_to url_for(:controller => :users, :action => :index)
   end
 
+  def feedback
+    @posts = Feedback.joins(:post).joins(:feedback_type).select('"posts"."id", "posts"."title", "feedbacks"."created_at", "feedback_types"."short_code", "feedback_types"."color"')
```

Comment:
* `joins` takes multiple arguments: `.joins(:post, :feedback_type)`
 * SQL identifiers should be quoted with backticks, not double quotes: `.select('`posts`.`id`, `posts`.`title`, ...')`
 * This variable should be called `@feedbacks` - it's a list of feedbacks, not a list of posts. View will need updating to match.